src/transformers/models/bert/modeling_bert.py

```diff
@@ -831,8 +896,14 @@ def forward(
 
         device = input_ids.device if input_ids is not None else inputs_embeds.device
 
+        # past_key_values_length
+        past_key_values_length = past_key_values[0][0].shape[2] if past_key_values is not None else 0
+
+        batch_size, seq_length = input_shape
+        real_seq_length = seq_length + past_key_values_length
```

Comment:
(nit) - don't think we need that line, I'd just write `seq_length + past_key_values_length` below